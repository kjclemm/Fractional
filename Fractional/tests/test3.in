10 10
2 3 0
8 1 11
9 15 28
1 30 7
25 25 4
23 22 22
27 23 16
7 18 9
29 16 21
18 24 24
-2 14 15 25 8 58
8 15 20 32 9 41
17 45 -4 46 17 52
9 24 3 57 5 51
-1 40 11 32 15 40
17 70 -2 56 9 68
10 58 11 30 15 46
6 53 11 52 17 45
-3 52 17 51 3 60
3 30 19 69 18 54
