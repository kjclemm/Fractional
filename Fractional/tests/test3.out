2
0
3
3
5
4
4
4
6
2
